docs/developing/troubleshooting.md

```diff
@@ -35,16 +40,20 @@ very likely you are seeing this error.
 If possible, configure the app to use a base URL of `''`, literally
 the empty string. Then it will send HTTP redirects without
 specifying a base URL. If that isn't possible, Sandstorm apps should
-look at the
+look at the `Host:` header for a base URL.
```

Comment:
Funny thing is, they should actually use `X-Sandstorm-Base-Url` instead, since `Host` doesn't specify a URL scheme (http vs. https). I guess. I need to verify that it really is `X-Sandstorm-Base-Url` specifically, though. I'll merge this as-is, but consider testing/changing that too if you want.